satpy/add_overlays_and_decorate_image.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    output_filename=f'seviri_{composite_name:s}_{local_scn.start_time:%Y%m%d_%H%M}_{areaid:s}.tif'
</pre><p>Thoughts on putting a space between the "=" for styling? And maybe instead of printing just doing "output_filename" and letting the notebook handle printing it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>